### ISSUE TITLE
Two texts in one manuscript to be represented by two different sigla

### ISSUE THREAD

**Issue by ppasedach** [chchch/upama#41]
Some manuscripts transmit both the root text as well as a commentary. Is there any way, apart from transcribing them into separate TEI files,  of assigning different sigla to the manuscript, depending on whether one collates the root text or the commentary?

**Comment by chchch** [comment]
Hmm... so if you have unique `@xml:id`s for each verse/paragraph, then it you should be able to collate it against manuscripts containing only the root text or only the commentary. It would still have the same siglum though. Is it important for there to be two different sigla?

**Comment by wujastyk** [comment]
It's not quite the same issue, but I thought I'd mention it, since it's a little bit similar.  We are collating against manuscripts and a printed edition.  For the LaTeX printout, we want two layers of variant notes, one for the MS variants, and a separate one for the edition variants.  So we just manipulated the LaTeX file, creating a \Dfootnote for the printed ed. variants and doing a careful search-and-replace. The results are good.  

The drawback of working on the LaTeX output file directly is that if you add or change data to the files at Saktumiva, you would have to do the LaTeX manipulation all over again.

**Comment by ppasedach** [comment]
Yes, right, that's somewhat related. In the LaTeX output I too would like to have the apparatus for the commentaries, and I currently have up to three commentaries that I'd want to edit together with the root text, in different registers.
